custom/enikshay/nikshay_datamigration/management/commands/create_enikshay_cases.py

```diff
@@ -0,0 +1,170 @@
+from django.core.management import BaseCommand
+from django.db import transaction
+
+from casexml.apps.case.const import CASE_INDEX_EXTENSION
+from casexml.apps.case.mock import CaseFactory, CaseStructure, CaseIndex
+from corehq.apps.locations.models import SQLLocation
+
+from custom.enikshay.nikshay_datamigration.models import PatientDetail, Outcome, Followup
+from dimagi.utils.decorators.memoized import memoized
+
+ENIKSHAY_DOMAIN = 'enikshay-np'
+
+
+class EnikshayCaseFactory(object):
+
+    patient_detail = None
+
+    def __init__(self, patient_detail, nikshay_code_to_location):
+        self.patient_detail = patient_detail
+        self.factory = CaseFactory(domain=ENIKSHAY_DOMAIN)
+        self.nikshay_code_to_location = nikshay_code_to_location
```

Comment:
is there a reason for this to be a constructor arg / member of the factory (as opposed to an independent function that handles)?